resources/default-config.kdl

```diff
@@ -40,6 +40,16 @@ input {
         // scroll-method "no-scroll"
     }
 
+    trackpoint {
+        // off
+        // natural-scroll
+        // accel-speed 0.2
+        // accel-profile "flat"
+        // scroll-method "on-button-down"
+        // scroll-button 273
```

Comment:
What's the 273 button?

Reply:
I found that it's BTN_RIGHT, is that consistent for every mouse?

Reply:
It seemsto be.

Source: https://github.com/torvalds/linux/blob/master/include/uapi/linux/input-event-codes.h